### ISSUE TITLE
Testing the SQL snippet

### ISSUE THREAD

**Issue by cresmet** [callmekatootie/carbonate#8]
SQL snippet:

```sql
select * from dual
```

**Comment by callmekatootie** [comment]
Trying to figure out why the image was not generated for the above... Tested locally but did not face any issue. Let's try that again shall we...

```sql
select * from dual;
```

**Comment by callmekatootie** [comment]
Trying again with debug statements

```sql
select * from dual;
```

**Comment by callmekatootie** [comment]
another test

```sql
select * from dual
```

**Comment by callmekatootie** [comment]
@cresmet Was able to determine the issue - Prettier failed.

You can only test code that babel parses since the default configuration for my repo uses babel as the parser. If you have installed the Action in your repo, then you can use the `prettier-parser` input for the Action to select a different code parser.

Basically, you can only test with javascript based code in this repo. You would need to install the Action in your own repo to test with a different parser - one that is based off the type of code in your project

**Comment by callmekatootie** [comment]
Alternatively, in your Action, set `use-prettier` to false - that will disable prettier and proceed to parse your code...

However, I will still take this as a bug. It should not have failed and instead should have skipped using Prettier...

**Comment by callmekatootie** [comment]
Ok. Fix is in. Testing now:

```sql
select * from dual;
```

**Comment by callmekatootie** [comment]
Fixed in `v1.0.1`

**Comment by callmekatootie** [comment]
Closing due to inactivity
